app/auth/create.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import { Text, TextInput, HelperText, Button } from 'react-native-paper'
+import { useRouter } from 'expo-router'
+import { useDispatch, useSelector } from 'react-redux'
+
+import AuthPage from '@components/AuthPage'
+import { createUser, selectIsLoggedIn } from '@store'
+
+const CreateAccount = () => {
+    const dispatch = useDispatch()
+    const router = useRouter()
+
+    const [fName, setFName] = useState('')
+    const [lName, setLName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [passwordConfirm, setPasswordConfirm] = useState('')
+    const [badEmail, setBadEmail] = useState(false)
+    const [passwordsMatch, setPasswordsMatch] = useState(true)
+
+    useEffect(() => {
+        const re = /^[\w-\\.]+@crimson.ua.edu$/
```

Comment:
I think we could also support `@ua.edu` emails. If you agree, then also change the TextInput label to say "UA Email" rather than "Crimson Email."

Reply:
@ua.edu emails are for professors, should they be able to log in? Do they get meal plans?

Reply:
They could certainly buy a meal using the app. Also, I got a @ua.edu email for my research job, so it's not just professors. Perhaps this is an @ardymarky question though.

Reply:
Oh also, let's avoid the lazy variable name `re` and say something like `emailRegex`. :)